docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -0,0 +1,192 @@
+| ----------------------------- | ------------------------------------------------------- | --------------------------------------------------------------- |
+| `formTitle`                   | `"Report a Bug"`                                        | The title at the top of the feedback form.                      |
+| `submitButtonLabel`           | `"Send Bug Report"`                                     | The label of the submit button used in the feedback form.       |
+| `cancelButtonLabel`           | `"Cancel"`                                              | The label of cancel buttons used in the feedback form.          |
+| `addScreenshotButtonLabel`    | `"Add a screenshot"`                                    | The label of the button to add a screenshot to the form.        |
+| `removeScreenshotButtonLabel` | `"Remove screenshot"`                                   | The label of the button to remove the screenshot from the form. |
+| `nameLabel`                   | `"Name"`                                                | The label of the name input field.                              |
+| `namePlaceholder`             | `"Your Name"`                                           | The placeholder for the name input field.                       |
+| `emailLabel`                  | `"Email"`                                               | The label of the email input field.                             |
+| `emailPlaceholder`            | `"your.email@example.org"`                              | The placeholder for the email input field.                      |
+| `isRequiredLabel`             | `"(required)"`                                          | The label shown next to an input field that is required.        |
+| `messageLabel`                | `"Description"`                                         | The label for the feedback description input field.             |
+| `messagePlaceholder`          | `"What's the bug? What did you expect?"`                | The placeholder for the feedback description input field.       |
+| `successMessageText`          | `"Thank you for your report!"`                          | The message displayed after a successful feedback submission.   |
+| `errorTitle`                  | `"Error"`                                               | The title of the error message dialog.                          |
+| `formError`                   | `"Please fill out all required fields."`                | Form validation error message.                                  |
+| `emailError`                  | `"Please enter a valid email address."`                 | Email validation error mesage.                                  |
+| `genericError`                | `"Unable to send feedback due to an unexpected error."` | The generic error message.                                      |
+
+Example of customization:
+
+```javascript
+feedbackIntegration({
+  buttonLabel: "Feedback",
```

Comment:
`buttonLabel` doesn't exist, maybe we can add `nameLabel` instead in the example.

I also replaced the duplicate `Send Feedback`, it's a good example, but I think it looks better without duplicates.

````suggestion
  nameLabel: "Full Name",
  submitButtonLabel: "Send",
  formTitle: "Give Feedback",
````